src/auth0/client.js

```diff
@@ -14,7 +17,9 @@ function getEntity(rsp) {
 // Warp around the ManagementClient and detect when requesting specific pages to return all
 export default function pagedClient(client) {
   client.pool = new PromisePoolExecutor({
-    concurrencyLimit: API_CONCURRENCY
+    concurrencyLimit: API_CONCURRENCY,
+    frequencyLimit: API_FREQUENCY_PER_SECOND,
+    frequencyWindow: 1000 // 1 sec
```

Comment:
here's the API reference
https://www.npmjs.com/package/promise-pool-executor#constructor